DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/tools/server2server/activate-server-to-server.md

In order to process direct credit card transactions, the following is required of the merchant:

1. Register for a [MultiSafepay Control](https://merchant.multisafepay.com/signup) account.

> 2. Make sure you are [PCI](/faq/risk-and-fraud/what-do-i-need-to-know-about-credit-card-payments) compliant and receive final approval by MultiSafepay Risk Team at <risk@multisafepay.com> to enable this feature.

Reviewer: ```suggestion
2. Make sure you are [PCI](/faq/risk-and-fraud/what-do-i-need-to-know-about-credit-card-payments) compliant. To request approval and enable this feature, email the Risk Team at <risk@multisafepay.com>
```

> 3. Your merchant file and performance of the company will undergo screening and if you have received approval, our Integration team will begin the integration of Server to Server.

Reviewer: ```suggestion
3. We will check your merchant file and company performance. If approved, the Integration Team will integrate Server to Server.
```

> 4. Integrate with MultiSafepay via JSON [API](/api/#server-to-server) following the instructions provided in this documentation after our Integration team have activated the feature for you.

Reviewer: ```suggestion
4. Once activated, integrate with MultiSafepay via JSON [API](/api/#server-to-server) following these instructions.
```

> 5. Credit card transactions with MultiSafepay accounts are by default [3D Secure](/faq/general/glossary/#3d-secure) enabled, in line with PSD2 regulations. If you operate outside of Europe, contact your account manager at <sales@multisafepay.com> to configure a more flexible setup.

Reviewer: ```suggestion
5. Credit card payments are [3D Secure](/faq/general/glossary/#3d-secure) enabled by default, in line with PSD2 regulations. If you operate outside of Europe and require a more flexible setup, email your account manager at <sales@multisafepay.com>
```
